pkg/skaffold/kubernetes/portforward/kubectl_forwarder.go

```diff
@@ -181,7 +184,7 @@ func (*KubectlForwarder) monitorErrorLogs(ctx context.Context, logs io.Reader, c
 				strings.Contains(s, "error upgrading connection") {
 				// kubectl is having an error. retry the command
 				logrus.Tracef("killing port forwarding %v", p)
-				if err := cmd.Terminate(); err != nil {
+				if err := cmd.Process.Kill(); err != nil {
```

Comment:
test that this doesn't regress https://github.com/GoogleContainerTools/skaffold/issues/3679#issuecomment-646897706